src/main/java/vg/civcraft/mc/civmodcore/util/Maths.java

```diff
@@ -0,0 +1,182 @@
+package vg.civcraft.mc.civmodcore.util;
+
+/**
+ *
+ */
+public class Maths {
```

Comment:
I'd be surprised if these didn't already exist in the Apache stuff shaded into MC. Doesn't really matter though